sphinxcontrib/openapi/openapi30.py

```diff
@@ -262,6 +263,27 @@ def _httpresource(endpoint, method, properties, render_examples):
         for line in param.get('description', '').splitlines():
             yield '{indent}{indent}{line}'.format(**locals())
 
+    # print request content
+    if render_request:
+        request_content = properties.get('requestBody', {}).get('content', {})
+        if request_content:
+            schema = request_content['application/json']['schema']
+            req_properties = json.dumps(schema['properties'], indent=2,
```

Comment:
Please correct me if I'm wrong, but this line will render a schema, not an actual example. Isn't it confusing?

Reply:
Yes, the proposed update doesn't touch the example section and adds one more section that displays the request body. The latter is necessary for a developer to know exactly this part of the API specification. An example doesn't show all properties, doesn't specify whether they are required or optional. You can only guess what type a property is, but if it is enum you cannot figure out the enum list.

Reply:
> An example doesn't show all properties, doesn't specify whether they are required or optional. You can only guess what type a property is, but if it is enum you cannot figure out the enum list.

That's a valid point. Thanks.